### PR TITLE
SEO-3353 - JS controls

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Renamed images for alt text.

Regards,
Sharly A.

### CODE REVIEW

File: js/Maps/Getting-Started.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started with JavaScript Map Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline js/Maps/Getting-Started.md:3]: JavaScript `Maps` control, check and change in all places

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/Maps/Getting-Started.md (hunk 1 above)
Comment (Sharly-Atieno) [reply js/Maps/Getting-Started.md]: Modified

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**